services/cargohold/Makefile

```diff
@@ -64,6 +64,10 @@ $(DEB_IT):
 i:
 	../integration.sh $(EXE_IT) -s $(NAME).integration.yaml -i ../integration.yaml
 
+.PHONY: i-aws
+i-aws:
+	INTEGRATION_USE_REAL_AWS=1 ../integration.sh $(EXE_IT) -s $(NAME).integration.yaml -i ../integration.yaml
```

Comment:
Did you intend to use `-s $(NAME).integration-aws.yaml` ? The script writes merges and writes to that file, but the Makefile doesn't use that. (same for all the Makefiles)

Reply:
Erm, yeah... rebase gone wrong :/ should be all green now